### PR TITLE
fix: clear wording error in numpy warmup

### PR DESCRIPTION
**Pull request by jbaremoney**
## Description
In the backpropagation section, there was a comment saying the code was computing the gradient of parameters a, b, c, d wrt the loss. This is flipped and causes confusion about the math of backprop.

Specifically, the original comment:

> “Backprop to compute gradients of a, b, c, d with respect to loss”

has been changed to:

> “Backprop to compute gradient of the loss with respect to parameters a, b, c, d”

## Checklist

- [x] The issue (if any) that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [x] At most one issue is addressed in this pull request
- [x] Labels from the issue (if any) that this PR is fixing are added to this pull request
- [x] No unnecessary issues are included into this pull request.

### CODE REVIEW

**Review by AlannaBurke — APPROVED** [review]

**Comment by jbaremoney** [comment]
Hi @AlannaBurke thanks for the review and approval...

It looks like this PR is approved and the CLA check is green, but the required build / pytorch_tutorial_build_manager check is still in an “expected / waiting for status” state.

Would you be able to remove the stale label and, when CI is healthy again, either re-run the docs build or merge this when convenient?

Thanks again for your help!
